src/akamai/js/akamaiHD_flash.js

```diff
@@ -316,7 +316,25 @@ require("../../../html5-common/js/utils/constants.js");
       if (loaded && !rewind) return;
       else {
         try {
-          this.callToFlash("load("+rewind+")");
```

Comment:
The flash side was never using that variable. But the flash side function was trying to find aspect ratio without any input - so I passed aspectRatio over from here instead of the unused value.